recipes/thorvg/all/conanfile.py

```diff
@@ -102,6 +108,11 @@ def validate(self):
                 f"{self.ref} requires C++{self._min_cppstd}, which your compiler does not support."
             )
 
+        if is_msvc(self) and self.settings.build_type == "Debug":
+            raise ConanInvalidConfiguration(
+                f"{self.ref} doesn't support debug build on MSVC."
+            )
```

Comment:
I think that we could report this because it looks like a bug. It was working for the previous versions and it's failing now.